packages/frontend/src/components/select_sports.tsx

```diff
@@ -0,0 +1,61 @@
+import { Sport } from "@esp-group-one/types";
+import { useState } from "react";
+import type { ChangeEvent } from "react";
+
+export function SelectSport() {
```

Comment:
We need to be able to get the current `Sport` selected, so we need to take in an `onChange` parameter